tests/RestServer/LanguageFeatureTest.cpp

```diff
@@ -36,22 +36,22 @@
 #include "Basics/files.h"
 #include "Basics/icu-helper.h"
 #include "IResearch/common.h"
+#include <locale.h>
 
 // -----------------------------------------------------------------------------
 // --SECTION--                                                 setup / tear-down
 // -----------------------------------------------------------------------------
 
 namespace {
-void checkCollatorSettings(const std::string& language,
-                           bool isDefaultLanguage) {
+void checkCollatorSettings(std::string_view language, bool isDefaultLanguage) {
   // Create collator with expected language
   UErrorCode status = U_ZERO_ERROR;
   icu::Collator* expectedColl = nullptr;
   if (language == "") {
     // get default collator for empty language
     expectedColl = icu::Collator::createInstance(status);
   } else {
-    icu::Locale locale(language.c_str());
+    icu::Locale locale(language.data());
```

Comment:
Is the value of this string_view guaranteed to be null-terminated?
Because std::string_views are not, so creating a locale from a raw data pointer here may read over the std::string_view's bounds.

Reply:
yes, `language` variables here are created like this:
```
constexpr std::string_view language2 = "sv";
```

So they are guaranteed to be null terminated.